docs/_posts/examples/3400-01-10-live-update-feature.html

```diff
@@ -0,0 +1,65 @@
+---
+layout: example
+category: example
+title: Update a feature in realtime
+description: Change an existing feature on your map in realtime by updating its data
+tags:
+  - sources
+---
+<div id='map'></div>
+
+<script>
+var map = new mapboxgl.Map({
+    container: 'map',
+    style: 'mapbox://styles/mapbox/satellite-v9',
+    zoom: 0
+});
+
+var url = '{{site.baseurl}}/assets/hike.geojson';
+map.on('load', function () {
+    var xhr = new XMLHttpRequest();
+    xhr.onreadystatechange = function() {
+        if (xhr.readyState == XMLHttpRequest.DONE) {
+            var data = JSON.parse(xhr.responseText);
+
+            // save full coordinate list for later
+            var coordinates = data.features[0].geometry.coordinates;
+
+            // start by showing just the first coordinate
+            data.features[0].geometry.coordinates = [coordinates[0]];
+
+            // add it to the map
+            map.addSource('trace', { type: 'geojson', data: data });
+            map.addLayer({
+                "id": "trace",
+                "type": "line",
+                "source": "trace",
+                "paint": {
+                    "line-color": "yellow",
+                    "line-opacity": 0.75,
+                    "line-width": 5
+                }
+            });
+
+            // setup the viewport
+            map.jumpTo({ 'center': [coordinates[0][0], coordinates[0][1]], 'zoom': 14 });
```

Comment:
Ah, I see. Possibilities:

* Leave as is.
* Remove the third coordinate from the source data.
* Allow three-element arrays in [`LngLat.convert`](https://github.com/mapbox/mapbox-gl-js/blob/d7da95bace87a06531e88b50b61b0d47f730db96/js/geo/lng_lat.js#L90-L90).

@lucaswoj Thoughts?